server/app.js

```diff
@@ -44,6 +45,7 @@ app.use((req, res, next) => {
 
 app.use("/auth", authRouter);
 app.use("/users", userRouter);
+app.use("/bookingrequests", bookingRequestRouter);
```

Comment:
```suggestion
app.use("/booking-requests", bookingRequestRouter);
```
Please use hyphenated case for route names

Reply:
resolved by 783b90ecbe1678b1b3df6d390464cf96bda4fdfc